released/packages/coq-deriving/coq-deriving.0.1.1/opam

```diff
@@ -0,0 +1,38 @@
+opam-version: "2.0"
+name: "coq-deriving"
+version: "0.1.1"
```

Comment:
```suggestion
```
These are already given by the file path.

Reply:
Done.